src/pages/Apps/__snapshots__/AppsPage.storybook.storyshot

```diff
@@ -202,6 +202,11 @@ exports[`Storyshots Apps Page Render the Apps page 1`] = `
         font-size: 0.8em;
         height: 80px;
       }
+      @media (max-width: 500px) {
```

Comment:
You should be using a constant here. I forget where this is but there is a constant for this.